invenio_rdm_records/records/mappings/v6/rdmrecords/drafts/draft-v4.0.0.json

```diff
@@ -300,6 +300,42 @@
                 }
               }
             },
+            "funding" : {
```

Comment:
A newer version (`v5.0.0`) should be created. It can be done after the feature is implemented.